lang/backend/src/erasure/ctx.rs

```diff
@@ -0,0 +1,43 @@
+use url::Url;
+
+use super::erasure_symbol_table::{GlobalErasureSymbolTable, Param};
+
+pub struct Ctx {
```

Comment:
This can be removed, it is currently only used in a debug assert (impossible error case).